ghost/members-api/lib/repositories/member.js

```diff
@@ -797,6 +797,8 @@ module.exports = class MemberRepository {
             subscription_id: subscription.id,
             status: subscription.status,
             cancel_at_period_end: subscription.cancel_at_period_end,
+            trial_start: subscription.trial_start || null,
+            trial_end: subscription.trial_end || null,
```

Comment:
I think these need updating?

Reply:
See above 😬

Reply:
🙈 I had wiring in another PR and missed it was also pushed here originally. Updated here now, thanks.